web/packages/teleport/src/Discover/Database/CreateDatabase/CreateDatabaseDialog.tsx

```diff
@@ -0,0 +1,132 @@
+/**
+ * Copyright 2023 Gravitational, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import {
+  Text,
+  Flex,
+  AnimatedProgressBar,
+  ButtonPrimary,
+  ButtonSecondary,
+} from 'design';
+import * as Icons from 'design/Icon';
+import Dialog, { DialogContent } from 'design/DialogConfirmation';
+
+import { Timeout } from 'teleport/Discover/Shared/Timeout';
+import { TextIcon } from 'teleport/Discover/Shared';
+
+import type { Attempt } from 'shared/hooks/useAttemptNext';
+
+export type Props = {
```

Comment:
WDYT about naming it `CreateDatabaseDialogProps`? 
Thanks to this, it is immediately clear what type you use:

```ts
const props: CreateDatabaseDialogProps = {
  pollTimeout: 8080000000,
  attempt: { status: 'processing' },
  retry: () => null,
  close: () => null,
  next: () => null,
  dbName: 'db-name',
};
```